docs/tutorial/multi_car_paint.ipynb

```diff
@@ -0,0 +1,541 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "i9nXiSl26DJK"
+   },
+   "source": [
+    "# Quantum Alternating Operator Ansatz for Two-Color Multi Car Paint Shop Problem\n",
```

Comment:
Please change this as QAOA because this is not Quantum Alternating Operator Ansatz.